### DISCUSSION TITLE
Athena on Windows 10

### DISCUSSION THREAD

**Discussion by andreangc19** [PrincetonUniversity/athena#520] [Q&A]
Hi

I am a beginner user of Athena and I have a Windows 10 machine. After testing the code I realized that I get many errors because some of the code in the tst folder was written for Linux. Is there any version of the test code that works on Windows? Does Athena only work on SO Linux?

**Comment by CatoDee** [comment]
I am not sure if anyone is running Athena++ on Windows...
But if you use Linux or Mac OS, things will be easier. There are many astrophysics simulations run on Linux servers. it is good to learn it early. 

If you have to use Windows 10 to run the test, the WSL (https://learn.microsoft.com/en-us/windows/wsl/install)  is the best choice.

**Comment by tomidakn** [comment]
Yes, I do recommend you to use WSL2 to run Athena++ on Windows. You can even use MPI once you install some packages.
